lib/grizzly/zwave/commands/failed_node_remove_status.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Grizzly.ZWave.Commands.FailedNodeRemoveStatus do
+  @moduledoc """
+  This command reports on the attempted removal of a presumed failed node.
+
+  Params:
+
+    * `:node_id` - the id of the node which removal for failure was attempted
+    * `:seq_number` - the sequence number of the removal command
+    * `:status` - whether the presumed failed node was removed
+
+  """
+
+  @behaviour Grizzly.ZWave.Command
+
+  alias Grizzly.ZWave.{Command, DecodeError}
+  alias Grizzly.ZWave.CommandClasses.NetworkManagementInclusion
+
+  @type status :: :done | :failed_node_not_found | :failed_node_remove_fail
+  @type param :: {:node_id, byte} | {:seq_number, ZWave.seq_number()} | {:status, status}
```

Comment:
I think you `alias Grizzly.ZWave` in this file you will fix the dialyzer warning.

Reply:
Fixed. Thanks!